src/generated/api/apikeys/v2/src/transport.rs

```diff
@@ -124,18 +152,15 @@ impl super::stub::ApiKeys for ApiKeys {
         options: gax::options::RequestOptions,
     ) -> Result<gax::response::Response<longrunning::model::Operation>> {
         let options = gax::options::internal::set_default_idempotency(options, false);
+        let path = format!("/v2/{}", {
+            &req.key
+                .as_ref()
+                .ok_or_else(|| gaxi::path_parameter::missing("key"))?
+                .name
```

Comment:
wouldn't we check if this is empty?

Also funny aside: the `name` field is annotated as `OUTPUT_ONLY` despite being part of the path. :upside_down_face: 

https://github.com/googleapis/googleapis/blob/6942862f40d7a65b8e65ec491605d90d48458eaa/google/api/apikeys/v2/resources.proto#L49